resources/testdata/glyphs2/Unicode-UnquotedHex.glyphs

```diff
@@ -0,0 +1,12 @@
+{
+familyName = WghtVar;
+fontMaster = ();
+glyphs = (
+	{
+		glyphname = name;
+		layers = ();
+		unicode = 03A2;
```

Comment:
how about choosing a hex number that could also be interpreted as a decimal one by a naive parser, e.g. one without A-F digits such as "1234"?

Reply:
Done